homeassistant/components/zha/strings.json

```
@@ -249,6 +249,13 @@
       "face_4": "With face 4 activated",
       "face_5": "With face 5 activated",
       "face_6": "With face 6 activated"
+    },
+    "extra_fields": {
+      "color": "Color",
+      "duration": "Duration",
```

What unit of time is this? Maybe we should add that like "Duration in seconds". Or alternatively use a description, but that would require to use a selector rather than `vol.All(vol.Coerce(int), vol.Range(1, 255))`

---

Indeed, it is in seconds.

```suggestion
      "duration": "Duration in seconds",
```